src/components/Tabs/__snapshots__/Tabs.test.tsx.snap

```diff
@@ -15,7 +15,7 @@ exports[`Tabs renders 1`] = `
 .pounce-0 {
   z-index: 1;
   cursor: pointer;
-  color: gray-50;
+  color: white-200;
```

Comment:
It's strange that this snapshot is rendering the actual name of the color rather than the hex code. 

I don't think we should block this PR for it, it's been doing that since before this PR